packages/block-library/src/navigation/use-navigation-menu.js

```diff
@@ -22,16 +22,10 @@ export default function useNavigationMenu( ref ) {
 			const rawNavigationMenu = ref
 				? getEntityRecord( ...navigationMenuSingleArgs )
 				: null;
-			let navigationMenu = ref
+			const navigationMenu = ref
 				? getEditedEntityRecord( ...navigationMenuSingleArgs )
 				: null;
 
-			// getEditedEntityRecord will return the post regardless of status.
-			// Therefore if the found post is not published then we should ignore it.
-			if ( navigationMenu?.status !== 'publish' ) {
-				navigationMenu = null;
-			}
-
```

Comment:
I removed this because the `wp_navigation` menus being created by `UncontrolledInnerBlocks` were initially being created as drafts on purpose. Though it looks like that was also removed.

Not sure what to do now 🤔

Reply:
Switched it back to how it is in trunk